react/src/components/AssetDetail/AssetDetail.tsx

```diff
@@ -72,25 +81,16 @@ const AssetDetail: React.FC<AssetModalProps> = ({
         <Button type="link" iconNameAfter="close" onClick={onClose}></Button>
       </div>
       <div className={styles.middleSection}>
-        {fileType ? (
-          <>
-            <Suspense fallback={<LoadingSpinner />}>
-              <div className={styles.assetContainer}>
-                <AssetRenderer type={fileType} source={featureSource} />
-              </div>
-            </Suspense>
-            <Button /*TODO Download Action */>Download</Button>
-          </>
-        ) : (
-          <>
-            <SectionMessage type="info">Feature has no asset.</SectionMessage>
-            {!isPublicView && (
-              <Button type="primary" /* TODO Add asset to a feature */>
-                Add asset from DesignSafe
-              </Button>
-            )}
-          </>
-        )}
+        <Suspense fallback={<LoadingSpinner />}>
+          <div className={styles.assetContainer}>
+            <AssetRenderer />
+          </div>
+          <AssetButton
```

Comment:
> Not sure that I love having the asset renderer declared and used in AssetDetail nor how the logic is laid out. Also open to suggestions on the AssetButton and what should be passed to it

 All I can think of is putting the buttons inside AssetDetail to avoid having a switch case twice on featureType (and keeping things together like how AssetButton has  preview.html of point cloud and then AssetPointCloud has index.html of point cloud). 
 
 but really, its fine as is.  If we want, we can rethink it when we get to [Questionnaire](https://tacc-main.atlassian.net/issues/WG-389). Right now, it works fine as is 👍 .